### ISSUE TITLE
Can't use string templates such as %(name)s in the DIR part of --logfile-format

### ISSUE THREAD

**Issue by casparvl** [easybuilders/easybuild-framework#2815]
If I use e.g.

```
--logfile-format="easybuild-%(name)s,%(name)s-%(version)s-%(date)s.%(time)s.ebout.log"
```

However, the %(name)s in the directory-part is not replaced. So, I get folders that are actually called `'easybuild-%(name)s'`.

The documentation claims that these string templates should be usable for both the filename and directory: 

> The logfile format configuration setting contains a tuple specifying a log directory name and a template log file name. In both of these values, using the following string templates is supported:

See: https://easybuild.readthedocs.io/en/latest/Configuration.html#logfile-format

**Comment by ocaisa** [comment]
I took a look at the source and while the docs state that it is supported, it actually is not. It would require significant changes near https://github.com/easybuilders/easybuild-framework/blob/develop/easybuild/tools/config.py#L590 to make it look more like https://github.com/easybuilders/easybuild-framework/blob/develop/easybuild/tools/config.py#L609
as well as other changes in the source.

@boegel What do you think? The easy option is to change the docs to reflect the source.

@casparvl Is this a really important feature to you? It looks like bit of work...

**Comment by boegel** [comment]
It wasn't trivial to fix, but not *that* bad either, see #2818

**Comment by boegel** [comment]
Fixed with #2818, thanks for reporting this @casparvl!

**Comment by casparvl** [comment]
Wow, that wasn't as small a change as I would have expected it to be. Nice job on the fix, thanks! :)
